### ISSUE TITLE
Rails Pathway: Provide descriptive link text (multiple assignees)

### ISSUE THREAD

**Issue by MaoShizhong** [TheOdinProject/curriculum#28290]
Links in our lessons should be sufficiently descriptive to improve accessibility. As per the [layout style guide](https://github.com/TheOdinProject/curriculum/blob/main/LAYOUT_STYLE_GUIDE.md#links), link text should both concisely describe its purpose (or where the link will redirect a user), and must not include the words `this` nor `here`.

## Guidance

- Comment on this issue to let us know you'd like to get a section assigned to you. You can also ask for a specific one.
- Please wait until you are assigned by a maintainer before starting to work on this issue to avoid multiple people working on the same lessons.
- When you have been assigned a section, please refer to the [suggested link text changes spreadsheet](https://docs.google.com/spreadsheets/d/1rM9tjPIFZbKoMfd6OlLnjOhddBe2bivllY8_Bg5aGqY/edit?usp=sharing) for instructions on what changes need to be made for your assigned lesson(s).
  - For most of these, you should just be able to copy and paste the suggested text to replace the original text shown in the spreadsheet (which may be more than just the link itself). Any additional steps/clarifications will be noted by the appropriate link(s).
  - You are welcome to use a different wording from a given suggestion if you find a better one. If you do, you must ensure the link text is consistent if there are multiple instances of the same href in the same lesson.
  - Line numbers for the files should be mostly accurate. It's possible that I missed an unrelated PR for a file and some numbers are a little off, but you can always search the file for the relevant text as necessary.
  - You are *not* required to fix any unrelated, preexisting linting errors such as list styles/indentation. Existing linting errors may cause certain tests to fail when you open a PR. You can ignore this and a maintainer will let you know if further action is required.
- Make sure to read the [general contributing guide](https://github.com/TheOdinProject/.github/blob/main/CONTRIBUTING.md) as well as the [curriculum repo contributing guide](https://github.com/TheOdinProject/curriculum/blob/main/CONTRIBUTING.md) before starting your first contribution.
- Please add the following to your PR in the Issues section: `Related to #28290`. Since this issue is divided into sections, **individual PRs should NOT INCLUDE the `closes` keyword next to an issue number.**

### Ruby

- [x] @SumonGFC [How This Course Will Work](https://www.theodinproject.com/lessons/ruby-how-this-course-will-work) && [Installing Ruby](https://www.theodinproject.com/lessons/ruby-installing-ruby) && [Basic Data Types](https://www.theodinproject.com/lessons/ruby-basic-data-types)
- [x] @CJKbarts [Methods](https://www.theodinproject.com/lessons/ruby-methods)
- [x] @TheSeagullDev [Debugging](https://www.theodinproject.com/lessons/ruby-debugging) && [Basic Enumerable Methods](https://www.theodinproject.com/lessons/ruby-basic-enumerable-methods) && [Predicate Enumerable Methods](https://www.theodinproject.com/lessons/ruby-predicate-enumerable-methods)
- [x] @davidumoru [Nested Collections](https://www.theodinproject.com/lessons/ruby-nested-collections) && [Project: Caesar Cipher](https://www.theodinproject.com/lessons/ruby-caesar-cipher)
- [x] @Preslav977 [Project: Bubble Sort](https://www.theodinproject.com/lessons/ruby-bubble-sort) && [Object Oriented Programming](https://www.theodinproject.com/lessons/ruby-object-oriented-programming) && [Project: Tic Tac Toe](https://www.theodinproject.com/lessons/ruby-tic-tac-toe) && [Files and Serialization](https://www.theodinproject.com/lessons/ruby-files-and-serialization)
- [x] @LAKSHYA201 [Pattern Matching](https://www.theodinproject.com/lessons/ruby-pattern-matching) && [Blocks](https://www.theodinproject.com/lessons/ruby-blocks) && [Project: Custom Enumerables](https://www.theodinproject.com/lessons/ruby-custom-enumerables)
- [x] @Grayvox [A Very Brief Intro to CS](https://www.theodinproject.com/lessons/ruby-a-very-brief-intro-to-cs) && [Recursive Methods](https://www.theodinproject.com/lessons/ruby-recursive-methods) && [Project: Recursion](https://www.theodinproject.com/lessons/ruby-recursion)
- [x] @Crn0 [Space Complexity](https://www.theodinproject.com/lessons/ruby-space-complexity) && [Project: Binary Search Trees](https://www.theodinproject.com/lessons/ruby-binary-search-trees) && [Project: Knights Travails](https://www.theodinproject.com/lessons/ruby-knights-travails)
- [x] @peter-bachowski [Common Data Structures](https://www.theodinproject.com/lessons/ruby-common-data-structures-and-algorithms) && [HashMap Data Structure](https://www.theodinproject.com/lessons/ruby-hashmap-data-structure)
- [x] @MidlifeMechanicus [Test Driven Development](https://www.theodinproject.com/lessons/ruby-test-driven-development) && [Project: Connect Four](https://www.theodinproject.com/lessons/ruby-connect-four) && [Project: Ruby Final Project](https://www.theodinproject.com/lessons/ruby-ruby-final-project)
- [x] @Peaches-xo [Introduction to RSpec](https://www.theodinproject.com/lessons/ruby-introduction-to-rspec) && [Conclusion](https://www.theodinproject.com/lessons/ruby-conclusion)

### Databases

- [x] @AhmedTheGreatest [Databases](https://www.theodinproject.com/lessons/databases) && [Project: SQL Zoo](https://www.theodinproject.com/lessons/databases-sql-zoo)

### Ruby on Rails

- [x] @yuliana-r [Project: Installing Rails](https://www.theodinproject.com/lessons/ruby-on-rails-installing-rails) && [A Railsy Web Refresher](https://www.theodinproject.com/lessons/ruby-on-rails-a-railsy-web-refresher)
- [x] @forge34 [Routing](https://www.theodinproject.com/lessons/ruby-on-rails-routing) && [Project: Blog App](https://www.theodinproject.com/lessons/ruby-on-rails-blog-app) && [Deployment](https://www.theodinproject.com/lessons/ruby-on-rails-deployment)
- [x] @jessiejalca [Form Basics](https://www.theodinproject.com/lessons/ruby-on-rails-form-basics) && [Sessions Cookies and Authentication](https://www.theodinproject.com/lessons/ruby-on-rails-sessions-cookies-and-authentication) && [Project: Members Only!](https://www.theodinproject.com/lessons/ruby-on-rails-members-only)
- [x] @looonnng [Installing PostgreSQL](https://www.theodinproject.com/lessons/ruby-on-rails-installing-postgresql) && [Project: Private Events](https://www.theodinproject.com/lessons/ruby-on-rails-private-events)
- [x] @clappy-j [Active Record Queries](https://www.theodinproject.com/lessons/ruby-on-rails-active-record-queries) && [Active Record Callbacks](https://www.theodinproject.com/lessons/ruby-on-rails-active-record-callbacks)
- [x] @dellaighted [Advanced Forms](https://www.theodinproject.com/lessons/ruby-on-rails-advanced-forms) && [Project: Flight Booker](https://www.theodinproject.com/lessons/ruby-on-rails-flight-booker)
- [x] @guskirb [APIs and Building Your Own](https://www.theodinproject.com/lessons/ruby-on-rails-apis-and-building-your-own)
- [x] @SeanAverS [Working with External APIs](https://www.theodinproject.com/lessons/ruby-on-rails-working-with-external-apis)
- [x] @Victor-M-Giraldo [Project: Kittens API](https://www.theodinproject.com/lessons/ruby-on-rails-kittens-api) && [Project: Flickr API](https://www.theodinproject.com/lessons/ruby-on-rails-flickr-api) && [JS Bundling](https://www.theodinproject.com/lessons/ruby-on-rails-js-bundling)
- [x] @thatsthat [Turbo](https://www.theodinproject.com/lessons/ruby-on-rails-turbo) && [Stimulus](https://www.theodinproject.com/lessons/ruby-on-rails-stimulus) && [Mailers](https://www.theodinproject.com/lessons/ruby-on-rails-mailers) && [Project: Sending Confirmation Emails](https://www.theodinproject.com/lessons/ruby-on-rails-sending-confirmation-emails)
- [x] @ItsTosh [Advanced Topics](https://www.theodinproject.com/lessons/ruby-on-rails-advanced-topics)
- [x] @mathdebate09 [Websockets and Action Cable](https://www.theodinproject.com/lessons/ruby-on-rails-websockets-and-actioncable) && [Basic Messaging App (actioncable_lesson.md)](https://github.com/TheOdinProject/curriculum/blob/main/ruby_on_rails/mailers_advanced_topics/actioncable_lesson.md) && [Project: Rails Final Project](https://www.theodinproject.com/lessons/ruby-on-rails-rails-final-project) && [Conclusion](https://www.theodinproject.com/lessons/ruby-on-rails-conclusion)

**Comment by SeanAverS** [comment]
Hi @MaoShizhong 

I am interested on working on this and would like be assigned the `Working with External APIS` section please 😄

**Comment by MaoShizhong** [comment]
Assigned!

**Comment by mathdebate09** [comment]
Hey Mao, can you assign me the section on [Websockets and Action Cable](https://www.theodinproject.com/lessons/ruby-on-rails-websockets-and-actioncable) && [Basic Messaging App (actioncable_lesson.md)](https://github.com/TheOdinProject/curriculum/blob/main/ruby_on_rails/mailers_advanced_topics/actioncable_lesson.md) && [Project: Rails Final Project](https://www.theodinproject.com/lessons/ruby-on-rails-rails-final-project) && [Conclusion](https://www.theodinproject.com/lessons/ruby-on-rails-conclusion) to me?

**Comment by MaoShizhong** [comment]
@mathdebate09 They're in the Ruby on Rails sheet. Just checked it and can see them there.
Can you check again and see if they're there?

**Comment by mathdebate09** [comment]
Ah silly me 😆, I can see them, you can assign it to me

**Comment by MaoShizhong** [comment]
All good - assigned

**Comment by SumonGFC** [comment]
Can assign me any of the Ruby ones.

**Comment by MaoShizhong** [comment]
@SumonGFC I've assigned you to [How This Course Will Work](https://www.theodinproject.com/lessons/ruby-how-this-course-will-work) && [Installing Ruby](https://www.theodinproject.com/lessons/ruby-installing-ruby) && [Basic Data Types](https://www.theodinproject.com/lessons/ruby-basic-data-types)

**Comment by SumonGFC** [comment]
@MaoShizhong Noob question: Single commit preferred? or one commit for each lesson?

**Comment by MaoShizhong** [comment]
> @MaoShizhong Noob question: Single commit preferred? or one commit for each lesson?

Ultimately, either is fine. Personally, if I was working on this myself, I'd make separate commits for each lesson because that's what feels most intuitive and maintainable to me.

**Comment by CJKbarts** [comment]
I'm interested in contributing to this. Could you please assign me to any of the ruby sections except the last 2?

**Comment by MaoShizhong** [comment]
@CJKbarts Assigned you [Methods](https://www.theodinproject.com/lessons/ruby-methods)

**Comment by CJKbarts** [comment]
Thank you
I'll start working on it today

**Comment by SumonGFC** [comment]
I can do another one if it is fine for us to do more than one.

**Comment by MaoShizhong** [comment]
> I can do another one if it is fine for us to do more than one.

Appreciate the enthusiasm. As with previous similar issues, to encourage wider participation, we're limiting to one per person for now. These changes aren't urgent, so we're not in a rush to get them done as soon as possible.

**Comment by TheSeagullDev** [comment]
I'm willing to take a section.

**Comment by AhmedTheGreatest** [comment]
I am also willing to get assigned.

**Comment by MaoShizhong** [comment]
@AhmedTheGreatest assigned you to [Databases](https://www.theodinproject.com/lessons/databases) && [Project: SQL Zoo](https://www.theodinproject.com/lessons/databases-sql-zoo)

@TheSeagullDev assigned you to [Debugging](https://www.theodinproject.com/lessons/ruby-debugging) && [Basic Enumerable Methods](https://www.theodinproject.com/lessons/ruby-basic-enumerable-methods) && [Predicate Enumerable Methods](https://www.theodinproject.com/lessons/ruby-predicate-enumerable-methods)

**Comment by davidumoru** [comment]
Hi @MaoShizhong 

I'd like to get a section assigned to me :)

**Comment by Grayvox** [comment]
Hey @MaoShizhong ! I'd love to help with the [A Very Brief Intro to CS](https://www.theodinproject.com/lessons/ruby-a-very-brief-intro-to-cs) && [Recursive Methods](https://www.theodinproject.com/lessons/ruby-recursive-methods) && [Project: Recursion](https://www.theodinproject.com/lessons/ruby-recursion) sections if that's alright.

**Comment by MaoShizhong** [comment]
@Grayvox Assigned you to those lessons.

@davidumoru Assigned you to [Nested Collections](https://www.theodinproject.com/lessons/ruby-nested-collections) && [Project: Caesar Cipher](https://www.theodinproject.com/lessons/ruby-caesar-cipher)

**Comment by CJKbarts** [comment]
@MaoShizhong could you please review my pull request?

I don't know if there's something I'm supposed to do...

**Comment by MaoShizhong** [comment]
@CJKbarts We're all volunteers here (including you!) where we all have our own lives and families etc. to get to. We will get to reviewing PRs when we are able to, even if that does take a little time, especially when a PR does not contain an urgent change. It may not even be me - anyone on the team may review a PR if appropriate.

Coincidentally, I did just finish reviewing your PR, but even if I hadn't, I or another maintainer would get to it eventually, so please be patient.

**Comment by CJKbarts** [comment]
I understand that...I'm sorry for rushing you.

When I read the documentation for pull requests it said to to request for a review from one of the reviewers of the PR but I didn't have you on there so I didn't know if you weren't able to do it because I hadn't added you on there...and I didn't know how.

I'm sorry once again.

**Comment by CJKbarts** [comment]
I reread what I typed and I dunno if it makes sense...buh long and short is that I was confused

It won't happen again

**Comment by MaoShizhong** [comment]
> I reread what I typed and I dunno if it makes sense...buh long and short is that I was confused
> 
> It won't happen again...thank you for your patience

If you have any uncertainties about your PR, you can always comment on the PR with whatever questions and when a maintainer has time to review it, they can address your concerns :)

**Comment by CJKbarts** [comment]
Thank you

**Comment by AhmedTheGreatest** [comment]
I have submitted a Pull Request

**Comment by Grayvox** [comment]
Just so you know - finished my assigned sections. :)

**Comment by yuliana-r** [comment]
@MaoShizhong happy to help with a section (or more) - I have no preference. Thank you :)

**Comment by MaoShizhong** [comment]
@yuliana-r assigned you to [Project: Installing Rails](https://www.theodinproject.com/lessons/ruby-on-rails-installing-rails) && [A Railsy Web Refresher](https://www.theodinproject.com/lessons/ruby-on-rails-a-railsy-web-refresher)

**Comment by peter-bachowski** [comment]
Hi Mao I'd like to be assigned a section please :smile:

**Comment by MaoShizhong** [comment]
@peter-bachowski assigned you to [Common Data Structures](https://www.theodinproject.com/lessons/ruby-common-data-structures-and-algorithms) && [HashMap Data Structure](https://www.theodinproject.com/lessons/ruby-hashmap-data-structure)

**Comment by dellaighted** [comment]
Hi @MaoShizhong I would like to work on Advanced Forms && Project: Flight Booker!

**Comment by MaoShizhong** [comment]
@dellaighted Assigned those lessons to you

**Comment by Peaches-xo** [comment]
Gday MaoMao! Can I please be assigned a section (Its my first time, Im so nervous)

**Comment by MaoShizhong** [comment]
Was wondering when you'd show up in the repos! @Peaches-xo assigned you to [Introduction to RSpec](https://www.theodinproject.com/lessons/ruby-introduction-to-rspec) && [Conclusion](https://www.theodinproject.com/lessons/ruby-conclusion)

**Comment by Preslav977** [comment]
Good day. I'd like to be assigned as well. I hope everything will go well.

I might have to read the contributing guide as well. I am nervous as well, but I want to try it.

**Comment by MaoShizhong** [comment]
@Preslav977 assigned you to [Project: Bubble Sort](https://www.theodinproject.com/lessons/ruby-bubble-sort) && [Object Oriented Programming](https://www.theodinproject.com/lessons/ruby-object-oriented-programming) && [Project: Tic Tac Toe](https://www.theodinproject.com/lessons/ruby-tic-tac-toe) && [Files and Serialization](https://www.theodinproject.com/lessons/ruby-files-and-serialization)

The changes are all detailed in the spreadsheet linked in the `Guidance` section above, and everything else will be in the contributing guide.
If you run into any issues, feel free to ask any questions here or in your PR (if you opened your PR).

**Comment by dellaighted** [comment]
Hi @MaoShizhong I was assigned the Advanced Forms && Project: Flight Booker lessons.
Just a minor question. In the Advanced Forms lesson assignment section (https://www.theodinproject.com/lessons/ruby-on-rails-advanced-forms#assignment), I have to fix number 4 according to the spreadsheet of fixes linked in the issue. I noticed that the current version of number 4 in the lesson is slightly different from the original text in the spreadsheet, but seems to match the replacement text already (maybe this was already fixed and the spreadsheet not updated? I'm not sure). Anyways the last two sentences of number 4 in the assignment is The example covers a lot of the things we’ve gone over so far, so follow along. Also note how he does the allowing of nested attributes in Rails 4. 
This part isn't included in the replacement text, so shall I go ahead and keep it or remove it?

**Comment by MaoShizhong** [comment]
> Hi @MaoShizhong I was assigned the Advanced Forms && Project: Flight Booker lessons. Just a minor question. In the Advanced Forms lesson assignment section (https://www.theodinproject.com/lessons/ruby-on-rails-advanced-forms#assignment), I have to fix number 4 according to the spreadsheet of fixes linked in the issue. I noticed that the current version of number 4 in the lesson is slightly different from the original text in the spreadsheet, but seems to match the replacement text already (maybe this was already fixed and the spreadsheet not updated? I'm not sure). Anyways the last two sentences of number 4 in the assignment is The example covers a lot of the things we’ve gone over so far, so follow along. Also note how he does the allowing of nested attributes in Rails 4. This part isn't included in the replacement text, so shall I go ahead and keep it or remove it?

@dellaighted 
Below is taken from the current deployed advanced forms lesson: 
![image](https://github.com/TheOdinProject/curriculum/assets/122839503/3a7bfc1a-f2ff-4a89-908f-858e69477db5)

Which as far as I can see, the first sentence matches the relevant row's original text, not the replacement text:
> Read [this blog post from Peter Rhoades](https://www.createdbypete.com/2014/04/04/working-with-nested-forms-and-a-many-to-many-association-in-rails-4.html) on working with nested forms.

The original text field only concerns the text to be replaced, so leave the last two sentences as they are, and only replace the original text with the replacement text. I'll probably rename the "original text" column to "text to replace" to make it clearer that the text doesn't necessarily include the entire line's contents.

**Comment by LAKSHYA201** [comment]
Hi @MaoShizhong , please assign me any of the lessons :)

**Comment by Preslav977** [comment]
Good day. I have a couple of questions. I made the changes, but I haven't filed a PR yet.

What I am trying to understand is if I should make the changes from the beginning because, in some places, the * in the knowledge checks were replaced with -, and I made sure to turn off my prettier code before starting to make the changes.

And I installed it in the clone repository, markdownlint-cli2. I don't know if that is a good thing or bad, because I already installed the extension and didn't see that I didn't need to do that.

So I don't know if I should start over again and just continue and make the necessary changes.

**Comment by MaoShizhong** [comment]
@LAKSHYA201 assigned you to [Pattern Matching](https://www.theodinproject.com/lessons/ruby-pattern-matching) && [Blocks](https://www.theodinproject.com/lessons/ruby-blocks) && [Project: Custom Enumerables](https://www.theodinproject.com/lessons/ruby-custom-enumerables)

**Comment by MaoShizhong** [comment]
@Preslav977 you don't need to start all over, just open the PR and we can go from there.

`*` changing to `-` without you manually changing them may be because you have `format on save` active, and your VSCode has markdown set to use the extension as its formatter, so turning off Prettier won't do anything here. I'd check that in `settings.json` to see if `[markdown]` has that set as its formatter.

Best advice I can give you is to get *very* familiar with `save without formatting` (can set or change its keyboard shortcut). You will not want any local format settings messing with any codebases that use different rules, so the best thing is to not run your IDE's formatters on them in the first place. That, or turn off `format on save` and bind `format` to its own keyboard shortcut (I personally prefer this). Then you can explicitly format only, and save only. Working on TOP stuff? Don't format, just save.

Likely there will be other lint errors to handle, so if there are in the PR review, I can provide instructions as appropriate there.

Ideally, you have both the extension and the package. The extension only allows you to view lint errors live without needing the CLI, or have it autoformat on save (but may not account for custom rules and fixers). The package allows you to use the CLI, which includes our NPM scripts.

**Comment by Preslav977** [comment]
Okay. I started over, and now everything seems fine.

Could I ask you if I need to delete my created branch because I followed the lesson using Git in the real world?

Because it is saying it is up to the main branch, and maybe I should open a pull request and continue.

**Comment by MaoShizhong** [comment]
> Okay. I started over, and now everything seems fine.
> 
> Could I ask you if I need to delete my created branch because I followed the lesson using Git in the real world?
> 
> Because it is saying it is up to the main branch, and maybe I should open a pull request and continue.

If you started over in a new branch, you can delete the old branch since that's not being used for anything. Just open your PR from your new branch with the fresh work.

**Comment by Preslav977** [comment]
Ah. Hold on. I am doing something wrong. I am trying to push, but I can't.

I want to push the changes from the branch I created and then merge them, but it says I don't have access rights.

And my branch is up-to date.

**Comment by MaoShizhong** [comment]
> Ah. Hold on. I am doing something wrong. I am trying to push, but I can't.
> 
> I want to push the changes from the branch I created and then merge them, but it says I don't have access rights.
> 
> And my branch is up-to date.

I can't see a fork of the curriculum repo in your repo list, so it's possible you just cloned TOP's repo and are now trying to push directly to it (which you can't).
You need to fork the repo, clone your fork, then branch and work from your fork. You can then open a PR from your fork's branch to TOP's main.

**Comment by Preslav977** [comment]
Okay. Thanks. This is harder than I thought.

**Comment by MaoShizhong** [comment]
> Okay. Thanks. This is harder than I thought.

It'll seem funky if you've never done it before, but after one or two times, it's a familiar process. I think a few steps in the contributing guide's instructions may have been missed or misinterpreted.

If you're super confused trying to fix this, just scrap it and delete your local clone. Start from fresh by following the steps for setup as I linked above, which starts with forking the curriculum repo and cloning your fork.

**Comment by Crn0** [comment]
Good day! I would like to contribute.

**Comment by MaoShizhong** [comment]
@Crn0 Assigned you to [Space Complexity](https://www.theodinproject.com/lessons/ruby-space-complexity) && [Project: Binary Search Trees](https://www.theodinproject.com/lessons/ruby-binary-search-trees) && [Project: Knights Travails](https://www.theodinproject.com/lessons/ruby-knights-travails)

**Comment by forge34** [comment]
Hello !
i would like to be assigned [Routing](https://www.theodinproject.com/lessons/ruby-on-rails-routing) && [Project: Blog App](https://www.theodinproject.com/lessons/ruby-on-rails-blog-app) && [Deployment](https://www.theodinproject.com/lessons/ruby-on-rails-deployment): if possible

**Comment by MaoShizhong** [comment]
@forge34 all yours

**Comment by looonnng** [comment]
Hi @MaoShizhong, I'd love to contribute and help out again! I'm flexible with any assignments.

**Comment by MaoShizhong** [comment]
@looonnng assigned you to [Installing PostgreSQL](https://www.theodinproject.com/lessons/ruby-on-rails-installing-postgresql) && [Project: Private Events](https://www.theodinproject.com/lessons/ruby-on-rails-private-events)

**Comment by SeanAverS** [comment]
Hi @MaoShizhong, If any more issues are open I would like to contribute again please. thanks 😄

**Comment by MaoShizhong** [comment]
@SeanAverS We'll still keep to one a person for now since it's not urgent and things are going at a good pace :) Appreciate the enthusiasm as always

**Comment by MidlifeMechanicus** [comment]
I'd be happy to take on some of the Ruby pages.

**Comment by MaoShizhong** [comment]
@MidlifeMechanicus assigned you to [Test Driven Development](https://www.theodinproject.com/lessons/ruby-test-driven-development) && [Project: Connect Four](https://www.theodinproject.com/lessons/ruby-connect-four) && [Project: Ruby Final Project](https://www.theodinproject.com/lessons/ruby-ruby-final-project)

**Comment by jessiejalca** [comment]
Hi! I'd love to take anything else that's needed. Maybe `Form Basics`?

**Comment by clappy-j** [comment]
Happy to be assigned any of them.

**Comment by MaoShizhong** [comment]
@jessiejalca assigned you to [Form Basics](https://www.theodinproject.com/lessons/ruby-on-rails-form-basics) && [Sessions Cookies and Authentication](https://www.theodinproject.com/lessons/ruby-on-rails-sessions-cookies-and-authentication) && [Project: Members Only!](https://www.theodinproject.com/lessons/ruby-on-rails-members-only)

@clappy-j assigned you to [Active Record Queries](https://www.theodinproject.com/lessons/ruby-on-rails-active-record-queries) && [Active Record Callbacks](https://www.theodinproject.com/lessons/ruby-on-rails-active-record-callbacks)

**Comment by guskirb** [comment]
I'd like to work on any of them if that's okay

**Comment by MaoShizhong** [comment]
Assigned you to [APIs and Building Your Own](https://www.theodinproject.com/lessons/ruby-on-rails-apis-and-building-your-own) @guskirb

**Comment by yuliana-r** [comment]
@MaoShizhong opened a PR for my sections, however it's failing the markdown lint tests - let me know if anything else is needed! Apologies for the delay.

**Comment by Victor-M-Giraldo** [comment]
I'm also open to working on any of these.

**Comment by thatsthat** [comment]
Hi, I'd like to contribute if there is any task still available

**Comment by ItsTosh** [comment]
Hi! I’d like to contribute if possible, I’m fine with any section.

**Comment by charu5673** [comment]
Hello! I'd like to contribute, if there's any sections left

**Comment by MaoShizhong** [comment]
@Victor-M-Giraldo assigned you to [Project: Kittens API](https://www.theodinproject.com/lessons/ruby-on-rails-kittens-api) && [Project: Flickr API](https://www.theodinproject.com/lessons/ruby-on-rails-flickr-api) && [JS Bundling](https://www.theodinproject.com/lessons/ruby-on-rails-js-bundling)

@thatsthat assigned you to [Turbo](https://www.theodinproject.com/lessons/ruby-on-rails-turbo) && [Stimulus](https://www.theodinproject.com/lessons/ruby-on-rails-stimulus) && [Mailers](https://www.theodinproject.com/lessons/ruby-on-rails-mailers) && [Project: Sending Confirmation Emails](https://www.theodinproject.com/lessons/ruby-on-rails-sending-confirmation-emails)

@ItsTosh assigned you to [Advanced Topics](https://www.theodinproject.com/lessons/ruby-on-rails-advanced-topics)

@charu5673 sorry, just assigned the last lesson group! Do take a look round other issues though, as there might be others that need help and may be suitable for you.

**Comment by MaoShizhong** [comment]
Thank you everyone who contributed to this :pray:
